### ISSUE TITLE
Usage when improper command

### ISSUE THREAD

**Issue by zendern** [sonatype-nexus-community/nancy#117]
* **What are you trying to do?**
Nancy should show usage when improper command is passed.

* **What feature or behavior is this required for?**
```
zender [projects/oss/nancy]  master ✔ → nancy thisaintright                                                                                        [a76019e]
open /Users/zender/.ossindex/.oss-index-config: no such file or directory
 __  __
/\ \/\ \
\ \ `\\ \      __       ___      ___    __  __
 \ \ , ` \   /'__`\   /' _ `\   /'___\ /\ \/\ \
  \ \ \`\ \ /\ \L\.\_ /\ \/\ \ /\ \__/ \ \ \_\ \
   \ \_\ \_\\ \__/.\_\\ \_\ \_\\ \____\ \/`____ \
    \/_/\/_/ \/__/\/_/ \/_/\/_/ \/____/  `/___/> \
                                            /\___/
                                            \/__/
  _        _                           _    _
 /_)      /_` _  _  _ _/_     _  _    (/   /_` _ . _  _   _/  _
/_) /_/  ._/ /_// //_|/  /_/ /_//_'  (_X  /   / / /_'/ //_/ _\
    _/                   _/ /
Nancy version: development
zender [projects/oss/nancy]  master ✔ →
```

* **How could we solve this issue? (Not knowing is okay!)**
I think when we introduced `nancy config` we only look for that value. We should tell users when they type it in wrong (cause I know i did :) ) what the correct usage is like we do when you type an incorrect flag. 

```
zender [projects/oss/nancy]  master ✔ → nancy --thisaintright                                                                                      [a76019e]
open /Users/zender/.ossindex/.oss-index-config: no such file or directory
flag provided but not defined: -thisaintright
Usage:
	go list -m all | nancy [options]
	go list -m all | nancy iq [options]
	nancy config
	nancy [options] </path/to/Gopkg.lock>
	nancy [options] </path/to/go.sum>

Options:
  -clean-cache
    	Deletes local cache directory
  -exclude-vulnerability value
    	Comma separated list of CVEs to exclude
  -exclude-vulnerability-file string
    	Path to a file containing newline separated CVEs to be excluded (default "./.nancy-ignore")
  -help
    	provides help text on how to use nancy
  -no-color
    	indicate output should not be colorized
  -output string
    	Styling for output format. ["json-pretty" "text" "csv" "json"] (default "text")
  -quiet
    	indicate output should contain only packages with vulnerabilities
  -token string
    	Specify OSS Index API token for request
  -user string
    	Specify OSS Index username for request
  -v	Set log level to Info
  -version
    	prints current nancy version
  -vv
    	Set log level to Debug
  -vvv
    	Set log level to Trace
```

* **Anything else?**

cc @bhamail / @DarthHater

**Comment by zendern** [comment]
@bhamail do you think #134 would clean this up too??

**Comment by bhamail** [comment]
> @bhamail do you think #134 would clean this up too??

@zendern Yes, good point. I do think the move to Cobra commands will fix this issue.

**Comment by deadlysyn** [comment]
Just confirming, hunting around for hacktoberfest ideas and it sounds like there is no work left to do here after #134 is completed?

**Comment by bhamail** [comment]
@deadlysyn Correct. I would say this is fixed by #161
